myna/application/exaca/meltpool.py

```diff
@@ -325,7 +331,7 @@ def merge_melt_times_with_rgb(
     points.SetData(vtk_points)
 
     # Add scalar data to vtk_dataset from df_merged
-    scalar_names = ["gid", "ts", "dts"]
+    scalar_names = ["Grain ID", "ts", "dts"]
```

Comment:
What are ts and dts here? Can you add a comment clarifying what these represent?

Reply:
I added a comment. Those are the solidification time and solidification time gradient. They were used to compute the melt pool boundaries from before that was a feature in ExaCA